### ISSUE TITLE
Unable to Register Hapi-OpenAPI Multiple Times

### ISSUE THREAD

**Issue by benpingilley** [krakenjs/hapi-openapi#120]
I am attempting to register `hapi-openapi` twice. I was able to do this with previous versions. I am now getting this error. Is it possible to do this post v17 hapi?

```
{ AssertionError [ERR_ASSERTION]: Plugin openapi already registered
          at internals.Server.register (/app/node_modules/hapi/lib/server.js:403:26)
          at Object.register (/app/modules/openapi2/index.js:10:22)
          at internals.Server.register (/app/node_modules/hapi/lib/server.js:419:35)
          at <anonymous>
     generatedMessage: false,
     generatedMessage: false,name: 'AssertionError [ERR_ASSERTION]',
     generatedMessage: false,code: 'ERR_ASSERTION',
     generatedMessage: false,actual: false,
     generatedMessage: false,expected: true,
     generatedMessage: false,operator: '==' }
```

**_server.js_**
```
const apiManifest = {
    server: { ... },
    register: {
        plugins: [
            { plugin: require('./modules/openapi1')  },
            { plugin: require('./modules/openapi2') },
        ]
    }
}

(async () => {
    try {
        const server = await Glue.compose(apiManifest)
        await server.start()
    } catch (err) {
        console.log(err)
    }
})();
```
**_./modules/openapi1/index.js_**
```
exports.plugin = {
    name: 'OpenAPI1',
    register: async (server, options) => {
        const swaggerizeHapi = require('hapi-openapi')
        await server.register({
            plugin: swaggerizeHapi,
            options: { ... }
        })
    }
}
```

**_./modules/openapi2/index.js_**
```
exports.plugin = {
    name: 'OpenAPI2',
    register: async (server, options) => {
        const swaggerizeHapi = require('hapi-openapi')
        await server.register({
            plugin: swaggerizeHapi,
            options: { ... }
        })
    }
}
```

**Comment by tlivings** [comment]
Previously hapi supported multiple connections. You can’t register the same plugin more than once per server (or connection) so I’d imagine that’s the issue you are seeing.

**Comment by benpingilley** [comment]
I posed the same [issue/question](https://github.com/hapijs/discuss/issues/652) to hapi and they said it's an issue with hapi-openapi. I needed to move forward so I merged the two swaggers into the same file so I would only call hapi-openapi once and to work around the issue. Thank you @tlivings.

**Comment by tlivings** [comment]
I will re-open this issue and investigate if that is the case.

**Comment by tlivings** [comment]
Looks like the fix would be to make `multiple: true` in `hapi-openapi`.

I don't really have a concern about adding that.

**Comment by benpingilley** [comment]
And that wouldn't cause issues?

**Comment by benpingilley** [comment]
If allowing `multiple: true`, we will need to change the [id](https://github.com/krakenjs/hapi-openapi/blob/master/lib/index.js#L125) to something dynamic since we can't have the same id multiple times. Any idea what the id should become?

**Comment by tlivings** [comment]
The main issues, I think, would be potential path conflicts.

**Comment by benpingilley** [comment]
Could me make the id equal the path?
This would make the id unique and error out if there are path conflicts.

``id: `apidocs-${spec.basePath}${docs.path}` ``

**Comment by tlivings** [comment]
One of the immediate issues that comes to mind is the root level docspath... But this value is overridable so it should be ok. 

As for `id` — that's an idea.

**Comment by tlivings** [comment]
fixed in 1.0.2.

**Comment by benpingilley** [comment]
Thank you!
